source/configuring_domain/03_managing_tracking/third_party_analytics.rst

```diff
@@ -2,66 +2,90 @@ Third-party analytics
 =====================
 
 The monitoring tools provided by Opendatasoft focus on API calls and usages.
-They are useful to gather precise information on how data is consumed (see :doc:`Analyzing domain usage </managing_domain/03_analyzing_domain_usage/index>`). But it can be limiting when trying to measure more broadly the audience of a portal: referral sources, visitors' geographical origin, time spent on the portal, etc.
+They are useful to gather precise information on :doc:`how data is consumed </managing_domain/03_analyzing_domain_usage/index>`.
+But, they can be limiting when trying to measure more broadly the audience of a portal.
 
-Third-party services, specialized in measuring audience, can give additional analytics such as those mentioned above. Opendatasoft supports integration with Google Analytics, XiTi, SmartTag by AT Internet, and Matomo.
-
-All third-party analytics integrations are activated and configured from **Configuration** > **Tracking** in the back office.
+Third-party services, specialized in measuring audience, can give additional analytics such as referral sources, visitors' geographical origin, or time spent on the portal.
 
+Opendatasoft supports integration with Google Analytics, XiTi, SmartTag by AT Internet, and Matomo.
 Once configured and running, they allow audience tracking on the front office of a domain.
 
 
-Google Analytics
-----------------
+Configuring Google Analytics
+----------------------------
 
 Google Analytics is an analytics solution edited by Google.
-
 Currently, only "Universal Analytics" properties are supported.
 
-To activate and configure Google Analytics:
+.. admonition:: Prerequisite
+    :class: important
+ 
+    Retrieve the Google Analytics ID for your domain. It should start with ``UA-``.
 
 1. In the **Configuration** > **Tracking** section of the back office, go to the Google Analytics area.
-2. Fill in the Google Analytics ID in the related text box (it should start with ``UA-``).
-3. Click on the Save button in the top right corner.
+2. Enter your Google Analytics ID in the corresponding field.
+3. Click on the **Save** button in the top right corner.
+
+Google Analytics is now activated on the domain.
```

Comment:
We could add something about GA asking for consent?

Reply:
Sure.
Here's my suggestion: "Using Google Analytics on your portal involves obtaining visitors' consent to comply with certain privacy laws such as the EU General Data Protection Regulation (GDPR). For more information, see https://help.opendatasoft.com/platform/en/configuring_domain/02_managing_legal_information/legals.html#cookie-banner."
What do you think?

Reply:
✅